docs/synchronizations-faq.md

```diff
@@ -0,0 +1,43 @@
+# Synchronizations FAQ
+##
+
+In Private Packagist synchronizations is how you keep packages, teams, their members, and access permissions in sync with your version control system like GitHub, GitLab or Bitbucket.
+
+You can find below answers for frequently-asked questions concerning synchronizations.
+
+## What happens if you promote a synchronization to a primary synchronization?
+
+Only with a primary synchronization that admins and owners teams will be added to Private Packagist.
```

Comment:
This sounds like an introduction to what is a primary sync. There seems to be something off at the beginning of the sentence `Only with a primary synchronization`. This sounds like there is a choice not have a primary synchronization. Also as we only call it a primary sync once there is more than one sync in an org. I would mention that as well to not confuse users that only have one sync setup e.g.

```
Private Packagist organizations that are synchronized with a remote organization have automatically all
admins and owners of the synchronization assigned to the admins and owners team in Private Packagist.
If an organization is synchronized with multiple remote organizations then only admins and owners of the
primary synchronization are added to the respective teams.
```